src/app/resources/resources-view.component.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { CouchService } from '../shared/couchdb.service';
+import { HttpModule } from '@angular/http';
+import { ResourcesViewComponent } from './resources-view.component';
+import { FormsModule }   from '@angular/forms'
+
+describe('ResourcesViewComponent', () => {
+    let component: ResourcesViewComponent;
+    let fixture: ComponentFixture<ResourcesViewComponent>;
+    let postSpy: any;
+    let getSpy: any;
+    let id: string;
+    let couchService;
+    let statusElement;
+    let testdata;
+    let testimage;
+    let testresource1;
+    //let resourceSrc;
+    let de;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+          imports: [ FormsModule, RouterTestingModule, HttpModule ],
+          declarations: [ ResourcesViewComponent ],
+          providers: [ CouchService ]
+        })
+        .compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(ResourcesViewComponent);
+        component = fixture.componentInstance;
+        de = fixture.debugElement;
+        couchService = fixture.debugElement.injector.get(CouchService);
+        testdata = { digest: "md5", length: 456321, revpos:1, stub:true };
+        testimage = { filename:"scenery.png", id: "scenery.png", mediaType: 'img', attachments:{ content_type: "application/image", testdata }};
```

Comment:
In your mock data attachments should be an object with one property the same name as the id and containing another object with content_type: 
`attachments: { 'scenery.png': { content_type: "application/image" } }`